### PR TITLE
Add webkitgtk 2.12.2

### PR DESCRIPTION
**Pull request by pyhalov**

### CODE REVIEW

File: components/library/webkitgtk/Makefile [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+REQUIRED_PACKAGES += library/zlib
+REQUIRED_PACKAGES += system/library/fontconfig
+REQUIRED_PACKAGES += system/library/freetype-2
+REQUIRED_PACKAGES += system/library/g++-4-runtime
+REQUIRED_PACKAGES += system/library/gcc-4-runtime
+REQUIRED_PACKAGES += system/library/math
+REQUIRED_PACKAGES += system/library
+REQUIRED_PACKAGES += x11/library/libx11
+REQUIRED_PACKAGES += x11/library/libxcomposite
+REQUIRED_PACKAGES += x11/library/libxdamage
```
Comment (alarcher) [inline components/library/webkitgtk/Makefile:85]: You need to add REQUIRED_PACKAGES += developer/gperf

**Comment by alarcher** [comment]
It is worth noting that pthread_attr_get_np was only added to illumos-gate on May 19, 2016.
Compilation will fail with anterior versions.

**Comment by pyhalov** [comment]
we have suitable version on build server

**Comment by alarcher** [comment]
I just mention that for potential testers.

File: components/library/webkitgtk/Makefile (hunk 1 above)
Comment (pyhalov) [inline components/library/webkitgtk/Makefile:85]: fixed

**Pull request merged**